client/src/actions/actionsUser/usersActions.js

```diff
@@ -170,5 +170,82 @@ export const fetchUser = (user_id) => {
       .catch(error => {
         dispatch(fetchUserFailure(error.message))
       })
+    }
   }
+
+  export const addAdmin = (name) => {
+  return function(dispatch, getState) {
+      dispatch(fetchUserRequest())
+      let base64 = require('base-64');
```

Comment:
instead require it here, we should use the import method. So, instead 
```
let base64 = require("base-64");
```

at the top of the file you can use 
```
import base64 from 'base-64'
```